internal/driver/daemon.go

```diff
@@ -6,13 +6,46 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/ory/analytics-go/v4"
+	"github.com/ory/x/healthx"
+	"github.com/ory/x/metricsx"
+	"github.com/spf13/cobra"
+
+	"github.com/ory/keto/internal/driver/config"
+
 	"github.com/ory/graceful"
 	"github.com/pkg/errors"
 	"github.com/soheilhy/cmux"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 )
 
+func (r *RegistryDefault) EnableSqa(cmd *cobra.Command) {
+	r.sqaService = metricsx.New(
+		cmd,
+		r.Logger(),
+		r.Config().Source(),
+		&metricsx.Options{
+			Service:       "ory-keto",
+			ClusterID:     metricsx.Hash(r.Config().DSN()),
+			IsDevelopment: strings.HasPrefix(r.Config().DSN(), "sqlite"),
+			WriteKey:      "qQlI6q8Q4WvkzTjKQSor4sHYOikHIvvi",
+			WhitelistedPaths: []string{
+				"/",
+				healthx.AliveCheckPath,
+				healthx.ReadyCheckPath,
+				healthx.VersionPath,
```

Comment:
Can you add the relation tuple paths here?

Reply:
Ah do I have to add all paths?

Reply:
Yes